Code/GraphMol/Resonance.cpp

```diff
@@ -0,0 +1,1355 @@
+// $Id$
+//
+//  Copyright (C) 2015 Paolo Tosco
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <iostream>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/MolOps.h>
+#include "Resonance.h"
+#include <boost/functional/hash.hpp>
```

Comment:
I'm not sure if it's still true, but the boost hashing code used to generate different results on 32 and 64 bit machines. It's probably not important here that you get consistent hashes across architectures, but if it is, you should probably switch to using the code in $RDBASE/Code/RDGeneral/hash (derived from an older version of boost.hash).